<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>time_freq_window</class>
 <widget class="QMainWindow" name="time_freq_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1496</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_layout" stretch="9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="9,1">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="3,7" columnstretch="7,2">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="ColorMeshPlot" name="plot_main" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item row="0" column="0">
             <widget class="SignalPlot" name="plot_top" native="true"/>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Signal X-Limits</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0,0,0,0" columnstretch="0,0">
               <item row="3" column="1">
                <widget class="QLineEdit" name="line_xlim2"/>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QPushButton" name="btn_refresh">
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLineEdit" name="line_xlim1"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Select data (scroll area)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="list_select_data"/>
              </item>
              <item>
               <widget class="QGroupBox" name="verticalGroupBox">
                <property name="title">
                 <string>Plot type</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QComboBox" name="combo_plot_type">
                   <item>
                    <property name="text">
                     <string>Raw harmonics</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Number of surrogates which harmonics higher than</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Harmonics relative to mean and std of surrogate distribution</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_6" rowstretch="1">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout_11" stretch="1,1,1,5">
                <item>
                 <widget class="QGroupBox" name="freqOptions">
                  <property name="title">
                   <string>Transform Options</string>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_14">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_4">
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QLabel" name="label_4">
                         <property name="frameShape">
                          <enum>QFrame::NoFrame</enum>
                         </property>
                         <property name="text">
                          <string>fmax</string>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QLineEdit" name="line_fmax">
                         <property name="maximumSize">
                          <size>
                           <width>76</width>
                           <height>22</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>5</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QLabel" name="label_3">
                         <property name="frameShape">
                          <enum>QFrame::NoFrame</enum>
                         </property>
                         <property name="text">
                          <string>fmin</string>
                         </property>
                        </widget>
                       </item>
                       <item alignment="Qt::AlignHCenter">
                        <widget class="QLineEdit" name="line_fmin">
                         <property name="minimumSize">
                          <size>
                           <width>70</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>70</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>0.025</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_15">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_16">
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="frameShape">
                          <enum>QFrame::NoFrame</enum>
                         </property>
                         <property name="text">
                          <string>Time resolution</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="line_res">
                         <property name="maximumSize">
                          <size>
                           <width>70</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>0.1</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_5"/>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_13">
                       <item>
                        <widget class="QLabel" name="label">
                         <property name="text">
                          <string>Sigma</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="line_sigma">
                         <property name="maximumSize">
                          <size>
                           <width>70</width>
                           <height>20</height>
                          </size>
                         </property>
                         <property name="text">
                          <string>1.05</string>
                         </property>
                         <property name="cursorPosition">
                          <number>0</number>
                         </property>
                         <property name="clearButtonEnabled">
                          <bool>false</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QGroupBox" name="verticalGroupBox">
                    <property name="title">
                     <string>Pre-processing</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_10">
                     <item>
                      <widget class="QRadioButton" name="radio_preproc_on">
                       <property name="text">
                        <string>On</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radio_preproc_off">
                       <property name="text">
                        <string>Off</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                     <string>Cut Edges</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_18">
                     <item row="0" column="0">
                      <widget class="QRadioButton" name="radio_cut_on">
                       <property name="text">
                        <string>On</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QRadioButton" name="radio_cut_off">
                       <property name="text">
                        <string>Off</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QFormLayout" name="formLayout_4">
                  <item row="1" column="1">
                   <widget class="QGroupBox" name="horizontalGroupBox">
                    <property name="title">
                     <string>Surrogate Count</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,1">
                     <item>
                      <widget class="QSlider" name="slider_surrogate_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="line_surrogate_2"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="PreprocessPlot" name="plot_preproc" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1">
           <property name="leftMargin">
            <number>6</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPlainTextEdit" name="text_log">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QPushButton" name="btn_calculate_all">
                 <property name="text">
                  <string>Transform All</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_calculate_single">
                 <property name="text">
                  <string>Transform Single</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="progress">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_progress">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1496</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignalPlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.SignalPlot</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorMeshPlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.ColorMeshPlot</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreprocessPlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.PreprocessPlot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
